File: 240613/두 숫자의 차의 최솟값/minimum-difference-between-two-numbers.py
N = int(input())
n_lst = list(map(int, input().split()))

min_diff = n_lst[-1] - n_lst[0]
for i in range(N):
    for j in range(i + 1, N):
        diff = n_lst[j] - n_lst[i]
        if diff < min_diff:
            min_diff = diff

print(min_diff)

File: 251120/두 수의 차의 최솟값/minimum-difference-between-two-numbers.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() 
{
    // Please write your code here.
    int N, min_diff, diff;
    cin >> N;
    vector<int> n_lst(N);

    for(int i = 0; i < N; i++)
        cin >> n_lst[i];

    min_diff = n_lst[N - 1] - n_lst[0];
    for(int i = 0; i < N; i++)
    {
        for(int j = i + 1; j < N; j++)
        {
            diff = n_lst[j] - n_lst[i];
            if(diff < min_diff)
                min_diff = diff;
        }
    }
    cout << min_diff;
    return 0;
}